components/event-processor/org.wso2.carbon.event.processor.core/src/main/java/org/wso2/carbon/event/processor/core/internal/util/EventProcessorUtil.java

```diff
@@ -289,10 +291,12 @@ public static AxisConfiguration getAxisConfiguration() {
             axisConfiguration = EventProcessorValueHolder.getConfigurationContext().
                     getServerConfigContext().getAxisConfiguration();
         } else {
-            axisConfiguration = TenantAxisUtils.getTenantAxisConfiguration(CarbonContext.
-                            getThreadLocalCarbonContext().getTenantDomain(),
-                    EventProcessorValueHolder.getConfigurationContext().
-                            getServerConfigContext());
+            ConfigurationContext configurationContext = EventProcessorValueHolder.getTenantConfig(PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantId());
+            if(configurationContext != null){
+                axisConfiguration = configurationContext.getAxisConfiguration();
+            }else{
+                throw new ExcecutionPlanRuntimeException("Tenant configuration not found");
```

Comment:
Thanks @ayash, done in pull request - https://github.com/wso2/carbon-event-processing/pull/160